Backpack/BarChart/Classes/BarChartCollectionView.swift

```diff
@@ -83,14 +83,18 @@ public final class BPKBarChartCollectionView: UICollectionView {
     lazy fileprivate var selectedMarker: UIView = {
         let view = UIView()
         view.backgroundColor = BPKColor.monteverde
+        view.layer.opacity = 0
         view.isHidden = true
         view.translatesAutoresizingMaskIntoConstraints = false
         return view
     }()
     
     fileprivate func updateSelectedMarkerPosition() {
+        var selectedBarTopPosition: CGFloat = selectedMarkerBottomConstraint.constant
+        var selectedBarOpacity: Float = 1.0
```

Comment:
the type is not inferred in this case? 🤔

Reply:
It infers type Double instead of float.
That said the CGFloat type above is inferred, so I've removed that